roles/libvirt_manager/tasks/ocp_layout.yml

```diff
@@ -30,7 +30,7 @@
       disk_file_name: "blank"
     _ocp_layout:
       vms:
-        ocp_master: >-
+        ocp: >-
```

Comment:
ok, so that will create "ocp" instead of "ocp_master" I think. I'm probably missing some brain cells to grasp the whole overview.

Reply:
Yep correct, this makes it so the VMs deployed via devscripts are the same name as the ones deployed from the overlay images. 

We only use one or the other at one time so there is no overlap and later code uses the vm name for things so it needs to be the same.